R/bootstrap.R

```diff
@@ -488,20 +489,33 @@ helpText <- function(...) {
 #' )
 #' @export
 #' @describeIn tabPanel Create a tab panel that can be included within a [tabsetPanel()] or a [navbarPage()].
-tabPanel <- function(title, ..., value = title, icon = NULL) {
+tabPanel <- function(title, ..., id = title, icon = NULL, value = NULL) {
```

Comment:
I think using `id` is too drastic. The change here will cause lots of messages to be printed for some users, and it will also add a HTML id to the DOM element, which will cause problems if the it happens to be the same ID as some other DOM element. It's not hard to imagine a case where the `value` is the same is as the `tabsetPanel`'s `id`.

The name `id` is also a bit misleading, since everywhere else that we use it, it's meant to set the DOM ID, but in this case, that's sort of a side effect of what we really want to do, which is provide a value when this tab is selected, if the `tabsetPanel` is an input.